packages/backend/src/docs/rest-api.json

```
@@ -1,1114 +1,1144 @@
+                      },
+                      "moodleNextFetchTimestamp": {
+                        "type": "number",
+                        "example": 1598865126
+                      },
+                      "moodleCurrentFetchInterval": {
+                        "type": "number",
+                        "example": 900000
+                      },
+                      "connectorsLength": {
+                        "type": "number",
+                        "example": 1598861227218
+                      },
+                      "connectorsActiveLength": {
+                        "type": "number",
+                        "example": 703550541160639500
+                      },
+                      "connectorsDefaultLength": {
+                        "type": "number",
+                        "example": "moodle-updates"
+                      }
+                    }
+                  },
+                  "error": {
                     "type": "array",
+                    "example": [],
                     "items": {
-                        "type": "object",
-                        "properties": {
-                            "message": {
-                                "type": "string"
-                            }
+                      "type": "object",
+                      "properties": {
+                        "message": {
+                          "type": "string"
                         }
+                      }
                     }
+                  }
                 }
+              }
+            },
+            "401": {
+              "$ref": "#/responses/401"
+            },
+            "500": {
+              "$ref": "#/responses/500"
             }
+          }
         }
-    },
-    "externalDocs": {
-        "description": "Get source code",
-        "url": "https://github.com/tjarbo/discord-moodle-bot/"
-    },
-    "responses": {
-        "400": {
-            "description": "Invalid Request! Common reason is that the request body does not meet the expected struture. Make shure to check the api documentation. More information are available within the HTTP response ",
-            "schema": {
+      },
+      "/verify": {
+        "get": {
+          "responses": {
+            "200": {
+              "description": "The provided token is valid!",
+              "schema": {
                 "$ref": "#/definitions/ApiResponse"
+              }
+            },
+            "401": {
+              "$ref": "#/responses/401"
+            },
+            "500": {
+              "$ref": "#/responses/500"
             }
-        },
-        "401": {
-            "description": "The authentification token is invalid or expired. Please login again to receive a new valid token!",
-            "schema": {
+          },
+          "summary": "Verify login status",
+          "description": "This endpoint allows to test if the peovided authentification token is valid or not",
```

```suggestion
          "description": "This endpoint allows to test if the provided authentification token is valid or not",
```
Not quite sure whether you want to keep the term `authentification`. Sounds weird to me, but since the tag is also called `Authentification`, at least it looks consistently weird :) We had this discussion before, hadn't we? :sweat_smile:

---

@p-fruck no, you are absolutly right! This is a good example why pr reviews are important. I will update everything